modules/vstudio/vs2005_fsproj.lua

```diff
@@ -0,0 +1,86 @@
+--
+-- vs2005_fsproj.lua
+-- Generate a Visual Studio 2005-2010 F# project.
+-- Copyright (c) 2009-2014 Jason Perkins and the Premake project
+--
+
+    local p = premake
+    p.vstudio.fs2005 = {}
+
+    local vstudio = p.vstudio
+    local fs2005 = p.vstudio.fs2005
+    local dotnetbase = p.vstudio.dotnetbase
+    local project = p.project
+    local config = p.config
+    local fileconfig = p.fileconfig
+    local dotnet = p.tools.dotnet
+
+    fs2005.elements = {}
+
+    fs2005.elements.project = function ()
+        return {
+            dotnetbase.xmlDeclaration,
+            dotnetbase.projectElement,
+            dotnetbase.commonProperties,
+            dotnetbase.projectProperties,
+            dotnetbase.configurations,
+            dotnetbase.applicationIcon,
+            dotnetbase.references
+        }
+    end
+
+    fs2005.elements.projectProperties = function ()
+        return {
+            dotnetbase.configurationCondition,
+            dotnetbase.platformCondition,
+            dotnetbase.schemaVersion,
+            dotnetbase.productVersion,
+            dotnetbase.projectGuid,
+            dotnetbase.outputType,
+            dotnetbase.appDesignerFolder,
+            dotnetbase.rootNamespace,
+            dotnetbase.assemblyName,
+            dotnetbase.targetFrameworkVersion,
+            dotnetbase.targetFrameworkProfile,
+            dotnetbase.projectTypeGuids
+        }
+    end
+
+    fs2005.elements.configuration = function ()
+        return {
+            dotnetbase.propertyGroup,
+            dotnetbase.debugProps,
+            dotnetbase.outputProps,
+            dotnetbase.compilerProps,
+            fs2005.tailCalls
+        }
+    end
+
+    function fs2005.generate(prj)
+        dotnetbase.prepare(fs2005)
+        dotnetbase.generate(prj)
+    end
+
+    function fs2005.tailCalls(cfg)
+        local tc = 'false'
+        if cfg.buildcfg == "Debug" then
+            tc = 'true'
+        end
+
+        _p(2, '<Tailcalls>%s</Tailcalls>', tc)
```

Comment:
Perhaps this could just be `_p(2, '<Tailcalls>%s</Tailcalls>', iif(cfg.buildcfg == "Debug", 'true', 'false')`, there's also a function to determine if the build is a debug build, perhaps it should use that instead of `cfg.buildcfg`?